Docs refactor (#421)
pydantic/logfire: docs/why-logfire/python-centric.md

# Python-centric insights :material-snake:

> Pydantic Logfire automatically instruments your code for minimal manual effort, provides exceptional insights into async code, offers detailed performance analytics, and displays Python objects the same as the interpreter. Pydantic Logfire  gives you a clearer view into how your Python is running than any other observability tool.

Reviewer: ```suggestion
Pydantic Logfire automatically instruments your code for minimal manual effort, provides exceptional insights into async code, offers detailed performance analytics, and displays Python objects the same as the interpreter. Pydantic Logfire gives you a clearer view into how your Python is running than any other observability tool.
```

Do we mean to consistently use **Pydantic Logfire**? Wasn't sure, had seen in some other places.

> 

Reviewer 2: ```suggestion
```

## Rich display of Python objects

![Logfire FastAPI screenshot](../images/logfire-screenshot-fastapi-arguments.png)

> In this example you can see the FastAPI arguments formatted as a Python object.

Reviewer 2: ```suggestion
In this example, you can see the parameters passed to a FastAPI endpoint formatted as a Python object.
```

> 

Reviewer 2: ```suggestion
```
